src/domain/alerts/contracts/decoders/__tests__/delay-modifier-decoder.helper.spec.ts

```diff
@@ -1,7 +1,7 @@
 import { toHex } from 'viem';
 import { faker } from '@faker-js/faker';
-import { DelayModifierDecoder } from '@/domain/alerts/contracts/delay-modifier-decoder.helper';
-import { transactionAddedEventBuilder } from '@/domain/alerts/contracts/__tests__/delay-modifier-encoder.builder';
+import { DelayModifierDecoder } from '@/domain/alerts/contracts/decoders/delay-modifier-decoder.helper';
+import { transactionAddedEventBuilder } from '@/domain/alerts/contracts/encoders/delay-modifier-encoder.builder';
```

Comment:
This should be placed in a `__tests__` folder, right?

My understanding is this is a class used only for testing (a builder), but if you think it could be used on other places we could leave it outside of test folders 🤔

Reply:
I moved the encoders in ca9b06462159ce6724ffcc41b3e0216e20d63932 into `__tests__`s folders as they are not required in the codebase. I left the decoders as is though because they are used by certain modules.